p2p/tests/kademlia.rs

```diff
@@ -402,7 +402,7 @@ fn bad_node_effects(
                 event_mapper_effect(store, a);
             }
             Action::Idle(_) => {
-                p2p_timeout_effects(store, &meta);
+                // p2p_timeout_effects(store, &meta);
```

Comment:
Is this a leftover comment? or something that should be enabled?

Reply:
I need to rewrite override functionality in p2p testing framework so I just left it here